exchanges/order/order_types.go

```diff
@@ -102,18 +101,19 @@ type SubmitResponse struct {
 	Pair      currency.Pair
 	AssetType asset.Item
 
-	ImmediateOrCancel    bool
-	FillOrKill           bool
+	TimeInForce          TimeInForce
 	PostOnly             bool
 	ReduceOnly           bool
 	Leverage             float64
 	Price                float64
-	AverageExecutedPrice float64
 	Amount               float64
 	QuoteAmount          float64
 	TriggerPrice         float64
 	ClientID             string
 	ClientOrderID        string
+	ImmediateOrCancel    bool
+	FillOrKill           bool
```

Comment:
Any reason to keep fields ImmediateOrCancel and FillOrKill with your new TimeInForce?

Reply:
and post only.